config/application.rb

```diff
@@ -29,5 +29,7 @@ class Application < Rails::Application
 
     # Don't generate system test files.
     config.generators.system_tests = nil
+
+    config.action_dispatch.perform_deep_munge = false
```

Comment:
Tell me more . . . on this line.

Reply:
Here is my source for formatting rails db to accept an attribute that has an array datatype with a default of an empty array: http://luxiyalu.com/rails-add-column-array-in-database/